src/app/DefaultAttributePersistenceProvider.h

```diff
@@ -34,16 +34,28 @@ class DefaultAttributePersistenceProvider : public AttributePersistenceProvider
 {
 public:
     // aStorage must outlive this object.
-    DefaultAttributePersistenceProvider(PersistentStorageDelegate & aStorage) : mStorage(aStorage) {}
+    DefaultAttributePersistenceProvider() {}
```

Comment:
I believe comment above needs update and move (to Init?)

Reply:
Done